guides/plugins/plugins/administration/using-base-components.md

```diff
@@ -0,0 +1,37 @@
+# Using base components
+
```

Comment:
You're starting very abrupt.
Please write an intro, like: The Shopware 6 Administration comes with a bunch of tailored Vue-components... or sth. like this